build_docker_images.sh

```diff
@@ -16,13 +57,15 @@ mkdir ${LIBDIR}
 cp ./stargate-lib/* $LIBDIR
 rm ${LIBDIR}/persistence*.jar
 
-#ls ${LIBDIR}
-
-docker build --target coordinator-4_0 --build-arg LIBDIR="$LIBDIR" -t stargateio/coordinator-4_0:$SGTAG .
-docker build --target coordinator-3_11 --build-arg LIBDIR="$LIBDIR" -t stargateio/coordinator-3_11:$SGTAG .
-docker build --target coordinator-dse-68 --build-arg LIBDIR="$LIBDIR" -t stargateio/coordinator-dse-68:$SGTAG .
+docker buildx build --target coordinator-4_0 --build-arg LIBDIR="$LIBDIR" -t stargateio/coordinator-4_0:$SGTAG $DOCKER_FLAGS .
```

Comment:
Specifying `buildx` causes Docker to use a different build engine from the default. You can read more about the differences in general in articles like [this](https://www.cloudsavvyit.com/12441/what-is-dockers-buildkit-and-why-does-it-matter/). 

But for our purposes here the most important thing is the ability to build images for multiple platforms. Note that the scripts in the `docker-images` repo (which this will eventually replace) uses `buildx`. See an example [here](https://github.com/stargate/docker-images/blob/master/cassandra-4.0/build.sh#L16).